.travis.yml

```diff
@@ -5,9 +5,9 @@ dist: jammy
 matrix:
   include:
     - python: 3.8
-    - python: 3.9
     - python: 3.10
     - python: 3.11
+    - python: 3.12
```

Comment:
It didn't seem right to build on FIVE versions of python so I removed python 3.9 so we would still be building on the current supported min version (3.8).